segmentation/utils/segmentation_utils.py

```diff
@@ -49,19 +49,17 @@ def compute_complete_expression_matrices(segmentation_labels, base_dir, tiff_dir
         # defined for mibitiff loading, we'll assume uniqueness for now but in the future we can address
         # different mibitiff loading techniques
         if is_mibitiff:
-            all_points = [mt_file for mt_file in os.listdir(tiff_dir) if mt_file.split(".")[1] in ["tif", "tiff"]]
-            points = [point.split(".")[0] for point in all_points]
+            all_points = io_utils.list_files(tiff_dir, substrs=['.tif'])
```

Comment:
I think this will also work on line 94?

Reply:
I think so; either way, I'll give it a try

Reply:
ooh, actually, line 94 is just filtering a pre-existing list of tifs, not scanning files in a directory.  `list_files` would still do the trick, but I think the list comp here is fine (although I think we can directly loop over batches instead of using a range?)

Reply:
okay, whatever you think is best